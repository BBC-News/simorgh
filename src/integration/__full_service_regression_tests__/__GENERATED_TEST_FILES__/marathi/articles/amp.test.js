/**
 * @service marathi
 * @pathname /marathi/articles/cp47g4myxz7o
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runCrossPlatformTests from '../../../../pages/articles/crossPlatformTests';
import runAmpTests from '../../../../pages/articles/ampTests';
describe(platform.toUpperCase(), () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runAmpTests();
  });
});
