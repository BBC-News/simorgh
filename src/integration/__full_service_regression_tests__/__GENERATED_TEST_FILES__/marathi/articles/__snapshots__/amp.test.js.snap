// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP  A11y I can see the skip to content link 1`] = `"थेट मजकुरावर जा"`;

exports[`AMP  AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
}
`;

exports[`AMP  AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP  Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/marathi"`;

exports[`AMP  Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही. बाह्य लिंक्सबद्दल आम्हाल काय वाटतं? इथे वाचा. बाह्य लिंक्सबद्दल आम्हाल काय वाटतं? इथे वाचा."`;

exports[`AMP  Header I can see the branding 1`] = `"BBC News, मराठी"`;

exports[`AMP  I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/9563/test/1b98b910-fcb3-11e9-9afe-1371928331c5.jpg"`;

exports[`AMP  I can see an image with a caption 2`] = `"फोटो कॅप्शन, ही चाचणी प्रतिमा, बीबीसी कॉपीराइट, फ्रान्सचा नकाशा दर्शविते. प्रतिमा पहिल्या तीन ब्लॉक्समध्ये आहे आणि त्यास हे मथळा आहे."`;

exports[`AMP  SEO Canonical link 1`] = `"http://localhost:7080/marathi/articles/cp47g4myxz7o"`;

exports[`AMP  SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP  SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP  SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP  SEO Heading level 1 1`] = `"ही या चाचणी लेखाची मथळा आहे"`;

exports[`AMP  SEO Lang attribute 1`] = `"mr"`;

exports[`AMP  SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png",
      "width": 1024,
    },
    "name": "BBC News मराठी",
    "noBylinesPolicy": "https://www.bbc.com/marathi/institutional-50418391#authorexpertise",
  },
  "dateModified": "2019-07-12T14:38:07.715Z",
  "datePublished": "2019-07-12T14:38:07.715Z",
  "headline": "चांद्रमोहीम : कसं होतं अपोलो 11 मिशन?",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/marathi/articles/cp47g4myxz7o",
    "@type": "WebPage",
    "name": "चांद्रमोहीम : कसं होतं अपोलो 11 मिशन?",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png",
      "width": 1024,
    },
    "name": "BBC News मराठी",
    "publishingPrinciples": "https://www.bbc.com/marathi/institutional-50418391",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png",
  "url": "http://localhost:7080/marathi/articles/cp47g4myxz7o",
}
`;

exports[`AMP  SEO OG description 1`] = `"चांद्रमोहीम : कसं होतं अपोलो 11 मिशन?"`;

exports[`AMP  SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png"`;

exports[`AMP  SEO OG image alt 1`] = `"BBC News मराठी"`;

exports[`AMP  SEO OG locale 1`] = `"mr-IN"`;

exports[`AMP  SEO OG site name 1`] = `"BBC News मराठी"`;

exports[`AMP  SEO OG title 1`] = `"चांद्रमोहीम : कसं होतं अपोलो 11 मिशन? - BBC News मराठी"`;

exports[`AMP  SEO OG type 1`] = `"article"`;

exports[`AMP  SEO OG url 1`] = `"http://localhost:7080/marathi/articles/cp47g4myxz7o"`;

exports[`AMP  SEO Page title 1`] = `"चांद्रमोहीम : कसं होतं अपोलो 11 मिशन? - BBC News मराठी"`;

exports[`AMP  SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP  SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP  SEO Twitter creator 1`] = `"@bbcnewsmarathi"`;

exports[`AMP  SEO Twitter description 1`] = `"चांद्रमोहीम : कसं होतं अपोलो 11 मिशन?"`;

exports[`AMP  SEO Twitter image alt 1`] = `"BBC News मराठी"`;

exports[`AMP  SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/marathi.png"`;

exports[`AMP  SEO Twitter site 1`] = `"@bbcnewsmarathi"`;

exports[`AMP  SEO Twitter title 1`] = `"चांद्रमोहीम : कसं होतं अपोलो 11 मिशन? - BBC News मराठी"`;
